src/ctype/CType.spec.ts

```diff
@@ -37,13 +29,6 @@ describe('CType', () => {
       },
       type: 'object',
     },
-    metadata: {
-      title: { default: 'CType Title' },
-      description: {},
-      properties: {
-        name: { title: { default: 'Name' } },
-      },
-    },
   } as ICType)
```

Comment:
As above as below, I have take out all the metadata from the test suites in the ctype spec files. I have created a new spec file for the metadata.